lib/resources/r4/coverage.yaml

```diff
@@ -15,6 +15,23 @@ fields:
   note: The resourceType must be Coverage.
   url: http://hl7.org/fhir/r4/json.html#resources
 
+- name: Coverage Encounter Extension
+  required: 'No'
+  cardinality: 0..1
+  type: Extension
+  description: Reference to the Encounter associated to the encounter level Coverage.
```

Comment:
If we're using Encounter-level Coverage, we should use it consistently.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/cf1c358e4663e9996a56a620adadf3a72acab2c7#diff-af9c966415db83d073f1bcec5ed6f875L22)